app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -82,148 +113,150 @@ const BigDelegatorOffer = () => (
 )
 
 interface Props {
-  updateStakePoolIdentifier: any
-  removeStakePool: any
-  stakePool: any
-  delegationFee: any
-  calculatingDelegationFee: any
+  stakePool: Stakepool
+  delegationFee: Lovelace
+  calculatingDelegationFee: boolean
   delegationValidationError: any
-  isShelleyCompatible: any
-  confirmTransaction: any
-  selectAdaliteStakepool: any
-  shouldShowConfirmTransactionDialog: any
-  txSuccessTab: any
+  isShelleyCompatible: boolean
+  shouldShowConfirmTransactionDialog: boolean
+  txSuccessTab: string
   gettingPoolInfo: boolean
-  poolRecommendation: any
-  pool: any
+  poolRecommendation: PoolRecommendation
+  pool: Stakepool
   isBigDelegator: boolean
+  validStakepools: Stakepools
   withAccordion: boolean
   title: string
+  updateStakePoolIdentifier: (poolHash: string) => void
+  confirmTransaction: (txConfirmType) => Promise<void>
+  selectAdaliteStakepool: () => void
 }
 
-class Delegate extends Component<Props, {dropShadow: boolean}> {
-  constructor(props) {
-    super(props)
-    this.state = {
-      dropShadow: false,
-    }
-  }
+const Delegate = ({
+  stakePool,
+  delegationFee,
+  calculatingDelegationFee,
+  delegationValidationError,
+  isShelleyCompatible,
+  shouldShowConfirmTransactionDialog,
+  txSuccessTab,
+  gettingPoolInfo,
+  poolRecommendation,
+  pool,
+  isBigDelegator,
+  validStakepools,
+  withAccordion,
+  title,
+  updateStakePoolIdentifier,
+  confirmTransaction,
+  selectAdaliteStakepool,
+}: Props): h.JSX.Element => {
+  if (!stakePool && ADALITE_CONFIG.ADALITE_STAKE_POOL_ID !== '') selectAdaliteStakepool()
```

Comment:
We should probably use `useEffect` here other wise the page wont re-render when changing tabs.

Reply:
Fixed